utils/bitcoin.ts

```diff
@@ -26,3 +26,13 @@ export const addressToScriptHash = (address: string) => {
   const scriptHash = Buffer.from(reverse(hash)).toString('hex');
   return scriptHash;
 };
+
+export const addMissingZerosToSatoshis = (value: number): string => {
+  const [integer, decimal] = value.toString().split('.');
+  const decimallWithMissingZeros = (decimal || '').padEnd(Math.log10(CONST.satoshiInBtc), '0');
+
+  return [integer, decimallWithMissingZeros].join('.');
+};
+
+export const getBtcvLabel = (value: number): string =>
```

Comment:
It actually doesn't return a label but formats the given amount to a specific format and then concats the current code at the end. `formatToBtcv` or `convertToBtcv` would be better name imo

Reply:
done